README.md

```diff
@@ -1,2 +1,51 @@
-# action-app-store-review
-WIP
+# VTEX app store review action
+> This project is a working in progress and is subject to breaking changes.
+
+This is a Github action that runs the [SonarScanner](https://docs.sonarqube.org/latest/analysis/scan/sonarscanner/) and send the generated analysis to VTEX SonarQube Server.
+
+## Usage
+
+Before creating your workflow, you need set two secret variables in your repository: The SonarQube server URL and your [SonarQube token](https://docs.sonarqube.org/latest/user-guide/user-token/).
+
+```yml
+name: Some workflow
+on:
+    pull_request:
+        branches: [master, main]
```

Comment:
Should we only run on PRs made for these two branches? There are cases of people maintaining code in branches like v0.x I think having a wildcard will not impact the main goal to serve vtex-reviews and can possibly benefit other teams :D

Reply:
Also, is it the best way to have this code in the readme? I don't know the answer yet

Reply:
Ah it seems like people are doing that way. Nevermind hahaha